src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/LanguageServerProjectSystem.cs

```diff
@@ -207,7 +214,26 @@ private async Task LoadOrReloadProjectAsync(string projectPath, ProjectBuildMana
                 {
                     foreach (var logItem in loadedFile.Log)
                     {
-                        _logger.LogWarning($"{logItem.Kind} while loading {logItem.ProjectFilePath}: {logItem.Message}");
+                        var projectName = Path.GetFileName(projectPath);
+                        var messageType = logItem.Kind switch
+                        {
+                            WorkspaceDiagnosticKind.Failure => LSP.MessageType.Error,
+                            WorkspaceDiagnosticKind.Warning => LSP.MessageType.Warning,
+                            _ => throw ExceptionUtilities.UnexpectedValue(logItem.Kind)
+                        };
```

Comment:
We have both this switch and then also the if () checking on it later...just inline these values into the calls to ShowToastNotificationAsync?